docs/source/api/v3/deliveryservice_requests_id_assign.rst

```diff
@@ -43,13 +43,12 @@ Request Structure
 	Content-Length: 28
 
 	{
-		"id": 1,
 		"assigneeId": 2
 	}
 
 Response Structure
 ------------------
-:assignee:        The username of the user to whom the :term:`Delivery Service Request` is assigned.
+:assigneeId:        The username of the user to whom the :term:`Delivery Service Request` is assigned.
```

Comment:
`assigneeId` needs to be added here, but `assignee` should not be removed as it is actually present in the response.